docs/baseband/vdif/index.rst

```diff
@@ -26,72 +26,59 @@ user-defined data.  The layout of these four words is specified by the file's
-confusion with Fourier channels).  A sequence of data frames all carrying the
-same (set of) sub-band(s) is called a **data thread**, denoted by its thread ID.
-A data set consisting of multiple concurrent threads is transmitted or stored
-as a serial sequence of frames called a **data stream**.  The collection of
-frames that cover all threads for a single time segment - equivalently, all
-thread IDs for the same header time and frame number - is a **dataframe set**
-(or just "frame set").
+A data frame may carry one or multiple frequency sub-bands or Fourier
+channels, and we refer to either of these as **channels** for short.  A sequence
+of data frames all carrying the same (set of) channels is called a **data
+thread**, denoted by its thread ID.  A data set consisting of multiple
+concurrent threads is transmitted or stored as a serial sequence of frames
+called a **data stream**.  The collection of frames that cover all threads for a
+single time segment - equivalently, all thread IDs for the same header time and
+frame number - is a **dataframe set** (or just "frame set").
 
 Strict time ordering of frames in the stream, while considered part of VDIF
 best practices, is not mandated, and cannot be guaranteed during data
 transmission over the internet.
 
 .. _vdif_usage:
 
-Usage
-=====
+Usage Notes
+===========
 
-The examples below use the small sample VDIF file :file:`baseband/data/sample.vdif`,
-and assumes the following modules have been imported::
+This section covers VDIF-specific features of Baseband.  Tutorials for general
+usage can be found under the :ref:`Using Baseband <using_baseband_toc>` section.
+The examples below use the small sample file ``baseband/data/sample.vdif``,
+and assume the `numpy` and `baseband.vdif` modules have been imported::
 
     >>> import numpy as np
     >>> from baseband import vdif
+    >>> from baseband.data import SAMPLE_VDIF
 
-Simple reading and writing of VDIF files can be done entirely using
-:func:`~baseband.vdif.open`, the master input/output function.
-
-Opening in binary mode provides a normal file reader but extended with
-methods to read a :class:`~baseband.vdif.VDIFFrame` (or
-:class:`~baseband.vdif.FrameSet`)::
+Baseband defines the :class:`~baseband.vdif.VDIFFrameSet` data container for
+storing a frame set as well as :class:`~baseband.vdif.VDIFFrame` one for storing
+a single frame. Opening in a VDIF file binary mode provides a file reader
+extended with methods to read both::
 
-    >>> from baseband.data import SAMPLE_VDIF
     >>> fh = vdif.open(SAMPLE_VDIF, 'rb')
     >>> fs = fh.read_frameset()
     >>> fs.data.shape
     (8, 20000, 1)
+    >>> fr = fh.read_frame()
+    >>> fr.data.shape
+    (20000, 1)
     >>> fh.close()
 
-Opening in stream mode wraps the low-level routines such that reading
-and writing is in units of samples.  It also provides access to header
-information::
-
-    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
-    >>> fh
```

Comment:
Again, I think we should keep this. For this file, it is actually expected to have been done below.

Reply:
I've retained it.